doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-test-bus-process.adoc

```diff
@@ -0,0 +1,201 @@
+|Verifies whether a process instance with the given `processInstanceId` is aborted. You can use this method if session persistence is enabled, otherwise use `assertProcessInstanceNotActive(long processInstanceId, KieSession ksession)`.
+
+|`assertNodeExists(ProcessInstance process, String... nodeNames)`
+|Verifies whether the specified process contains the given nodes.
+
+|`assertNodeActive(long processInstanceId, KieSession ksession, String... name)`
+|Verifies whether a process instance with the given `processInstanceId` contains at least one active node with the specified node names.
+
+|`assertNodeTriggered(long processInstanceId, String... nodeNames)`
+|Verifies whether a node instance is triggered for each given node during the execution of the specified process instance.
+
+|`assertProcessVarExists(ProcessInstance process, String... processVarNames)`
+|Verifies whether the given process contains the specified process variables.
+
+|`assertProcessNameEquals(ProcessInstance process, String name)`
+|Verifies whether the given name matches the specified process name.
+
+|`assertVersionEquals(ProcessInstance process, String version)`
+|Verifies whether the given process version matches the specified process version.
+
+|===
+--
+
+* Helper methods: Use following methods to create a new `RuntimeManager` and `RuntimeEngine` for a given set of processes with or without using persistence. For more information about persistence, see {URL_DEVELOPING_PROCESS_SERVICES}#assembly-process-engine[_{PROCESS_ENGINE_DOC}_].
++
+--
+.`RuntimeManager` and `RuntimeEngine` Management Methods
+[cols="30%,70%", options="header"]
+|===
+|Method
+|Description
+
+|`setupPoolingDataSource`
+|Configures a data source.
+
+|`getDs`
+|Returns the configured data source.
+
+|`getEmf`
+|Returns the configured `EntityManagerFactory`.
+
+|`getTestWorkItemHandler`
+|Returns a test work item handler that can be registered in addition to the default work item handler.
+
+|`clearHistory`
+|Clears the history log.
+
+|===
+--
+
+The following example contains a start event, a script task, and an end event. The example JUnit test creates a new session, starts the `hello.bpmn` process, and verifies whether the process instance is completed and the `StartProcess`, `Hello`, and `EndProcess` nodes are executed.
+
+.Example JUnit Test of `hello.bpmn` Process
+image::BPMN2/bpmn-example-junit-test-process.png[Example JUnit Test Process]
+
+[source,java]
+----
+public class ProcessPersistenceTest extends JbpmJUnitBaseTestCase {
+
+    public ProcessPersistenceTest() {
+        super(true, true);
+    }
+
+    @Test
+    public void testProcess() {
+
+        createRuntimeManager("hello.bpmn");
+
+        RuntimeEngine runtimeEngine = getRuntimeEngine();
+
+        KieSession ksession = runtimeEngine.getKieSession();
+
+        ProcessInstance processInstance = ksession.startProcess("com.sample.bpmn.hello");
+
+        assertProcessInstanceCompleted(processInstance.getId(), ksession);
```

Comment:
Change this to `assertProcessInstanceNotActive(processInstance.getId(), ksession);` please. The current method is [deprecated](https://github.com/kiegroup/jbpm/blob/06f1719c5fcaa8e0c6bb8f464a5dc4f8143e8bc4/jbpm-test/src/main/java/org/jbpm/test/JbpmJUnitBaseTestCase.java#L601-L609).

Reply:
Done